components/googleMap/googleMap.scss

```diff
@@ -0,0 +1,26 @@
+.ui-google-map__custom-marker {
```

Comment:
not everywhere https://github.com/Travix-International/travix-ui-kit/blob/7f5d9bfdb31a0cae9550a2d4a370a5f170d42a49/components/index.scss#L6

we're possible will move out of SCSS, so if you don't use any significant features — why not to start use plain CSS just now?